redhat/frr.spec.in

```diff
@@ -151,6 +151,7 @@ BuildRequires:  json-c-devel bison >= 2.7 flex make
 BuildRequires:  python27-devel python27-sphinx
 %else
 BuildRequires:  python-devel >= 2.7 python-sphinx
+BuildRequires:  c-ares-devel texinfo
```

Comment:
I believe this line should go under line 148, not inside this `if/else` block. The `if/else` block is only for properly versioning `python-sphinx`.

Reply:
Thanks for the review.  I believe the else is for RHEL and RHEL < 7 ?  I haven't tested the patch for RHEL6 (or any other rpm distro), so I was a little leery about putting it in the generic spec bits.  What do you think?  Should I test on RHEL6 too and fix that up too?  Any other distros I should try? Should I just push it and hope the CI/CD sorts it out? Or, is it good enough to just get RHEL7 working and move it up if/when it is tested on other distros?

Reply:
If it's convenient for you to test on Centos6 that would be very much appreciated and would help us get this in faster. Taking a brief look at the Centos6 RPM listings it looks like both of these packages have the same name there, and we certainly depend on them on that platform.

If you don't want to test on Centos6 -- @mwinter-osr, can you take a look at this?

Reply:
Sure, I'll try CentOS 6, should just a few minutes...

Reply:
CentOS 6 fails with:

configure.ac:60: require Automake 1.12, but have 1.11.1
autoreconf: automake failed with exit status: 1

I guess it doesn't build in a chroot now either and needs some manual hand-holding so may not be easy to build in a chroot...

I did try to load those packages in CentOS 6, so, at least, putting them in the generic section should not make things worse.  I'll re-test the patch in CentOS 7 the way you suggested and re-push...

Reply:
Thank you :)